docs/_tutorials/messagevpn-with-queue-java.md

```diff
@@ -0,0 +1,246 @@
+
+![]({{ site.baseurl }}/images/message-vpn-semp-objects.png)
+
+Clients can connect to a message-VPN and use its resources after proper authentication and authorization, controlled by the properties of following SEMP objects on the Solace message router:
+ 
+* The [Message-VPN]({{ site.docs-msg-vpn }}){:target="_top"} object defines the type and details of client authentication applied and restrictions to the combined resource usage of all VPN clients. 
+* A [Client Profile]({{ site.docs-client-profile }}){:target="_top"} object within a message-VPN defines resource usage restrictions applied to individual clients.
+* An [ACL (Access Control List) Profile]({{ site.docs-acl-profile }}){:target="_top"} object within a message-VPN defines access restrictions by listing allowance or denial of which clients can connect and to which topics and queues.
+* A [Client Username]({{ site.client-username }}){:target="_top"} object within a message-VPN has an associated Client Profile and an ACL Profile. The username provided by the connecting client will be the Client Username applied to that connection. The password property will be used if internal client authentication type has been specified for the message-VPN.
+
+Once connected, clients can send messages to message-VPN endpoints, represented as [Queue]({{ site.docs-queues }}){:target="_top"} or [Topic Endpoint]({{ site.docs-dtes }}){:target="_top"} SEMP objects within a message-VPN.
+
+You can learn more about above SEMP objects by following the provided links to the Solace documentation.
+
+## Building blocks using the SEMP Java Client Library
+
+Now we can start to implement the tasks as building blocks for the sample management tool.
+
+### Create a new message-VPN
+
+The first task is to create a new message-VPN with a name. Consulting the Solace documentation by following above [Message-VPN]({{ site.docs-msg-vpn }}){:target="_top"} link, we determine that it shall be configured for basic internal authentication, have storage size increased for persistent message queues from default 0 and have it enabled. Tip: the [SolAdmin]({{ site.docs-soladmin-home }}){:target="_top"} management GUI tool is using the same SEMP interface to the Solace message router. It will also show default values for new created objects. If in doubt, try the intended management operation using SolAdmin, which can be [downloaded here]({{ site.docs-solace-downloads }}){:target="_top"}.
```

Comment:
Comment on SolAdmin is inaccurate for SEMPv2. This will be accurate in the next generation of that UI.